decidim-core/app/models/decidim/user_group.rb

```diff
@@ -13,15 +13,16 @@ class UserGroup < ApplicationRecord
     mount_uploader :avatar, Decidim::AvatarUploader
 
     scope :verified, -> { where.not(verified_at: nil) }
-
-    # Public: Mark the user group as verified
-    def verify!
-      update_attribute(:verified_at, Time.current)
-    end
+    scope :rejected, -> { where.not(rejected_at: nil) }
```

Comment:
Can we add a validation so a user group can't be verified & rejected at the same time?